pkg/virt-controller/watch/migration_test.go

```diff
@@ -578,6 +580,13 @@ var _ = Describe("Migration watcher", func() {
 					Name: "compute", State: k8sv1.ContainerState{Running: &k8sv1.ContainerStateRunning{}},
 				}}
 
+				expectedRequestsLabel := targetPod.Spec.Containers[0].Resources.Requests.Memory().DeepCopy()
+				if hugepages != nil {
+					resourceName := k8sv1.ResourceName(k8sv1.ResourceHugePagesPrefix + hugepages.PageSize)
+					targetPod.Spec.Containers[0].Resources.Requests[resourceName] = guestMemory
+					expectedRequestsLabel.Add(guestMemory)
+				}
```

Comment:
Nit: Please move this to the argument of the `func`

Reply:
done.